src/mono/netcore/sample/wasm/WasmSample.csproj

```diff
@@ -39,11 +39,18 @@
       MicrosoftNetCoreAppRuntimePackDir="$(MicrosoftNetCoreAppRuntimePackDir)"
       MainAssembly="bin\WasmSample.dll"
       MainJS="$(MonoProjectRoot)wasm\runtime-test.js"
-      AssemblySearchPaths="@(AssemblySearchPaths)"/>
+      AssemblySearchPaths="@(AssemblySearchPaths)"
+      FilesToIncludeInFileSystem="@(WasmFilesToIncludeInFileSystem)"
+   />
     <Exec Command="chmod a+x $(AppDir)/run-v8.sh" />
   </Target>
 
+  <ItemGroup>
+    <PackageReference Include="System.Runtime.TestData" Version="$(SystemRuntimeTestDataVersion)" />
```

Comment:
Including System.Runtime.TestData does not make much sense for WasmSample

Reply:
Sounds good. I will just revert the sample to the original.